app/views/ferns/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<h3>
+  <%= @fern.name %>'s Fern
+</h3>
+
+<div id="disclaimer-warning">
+  <b>Disclaimer:</b> Interaction analysis is done by a machine learning model. It is not perfect.
+  Have fun, water your fern, and don't take it too seriously.<br>
+</div>
+
+<h4>Add Interaction</h4>
+<%= form_with url: fern_path(@fern.id), method: :patch, local: true do |f| %>
+  <%= f.label :interaction %>
+  <%= f.text_area :interaction, rows: 8 %>
```

Comment:
we'll need to double check with BE to make sure this is how they want the params labeled.